doc/decisions/database_plant_hierarchy.md

```diff
@@ -0,0 +1,133 @@
+# Database Plant Hierarchy
+
+## Problem
+
+The challenge is to define the structure of the database in a way that the data:
+
+- can be easily queried so the most specific data according the taxonomy is retrieved and
+- updated without redundancy (e.g. setting attributes for a whole family).
+
+Here we will discuss how the hierarchy should be defined in the database.
+
+## Constraints
+
+- The database structure should represent the simplified taxonomy we need for PermaplanT.
+- There must be a unique identifier for later updates of the database.
+
+## Assumptions
+
+- Performance of the database is efficient enough.
+- Subfamilies will not be introduced by PFAF/Permapeople.
+- A new plant added by us won't need subfamilies.
+- We don't need convar between species and variety.
```

Comment:
Should we specify here what is `convar` without adding it to the glossary?

Reply:
We focus on the glossary what we need and I think it is already enough :wink: